coderd/database/migrations/000223_notifications_metrics.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE notification_messages
+ADD COLUMN queued_seconds FLOAT NULL;
```

Comment:
Nit: The name `queued_seconds` implies an integer, consider changing the type or the name.

Reply:
I don't think it necessarily implies integer; `time.Duration`'s [`Seconds()`](https://pkg.go.dev/time#Duration.Seconds) returns a float64.

Reply:
Fair enough 😄.

In the DB, usually it means seconds:

```
    lifetime_seconds bigint DEFAULT 86400 NOT NULL,
    timeout_seconds integer NOT NULL
    connection_timeout_seconds integer DEFAULT 0 NOT NULL,
```

But fine to keep as float if we care about the precision.

Reply:
I suppose I don't strictly _need_ to keep sub-second precision on the length of time messages were queued for.
I'll switch to `integer` 👍

Reply:
This is backing a Prometheus metric, which are always floats, and by convention end in the SI unit, which for time is seconds.  It implies no precision.

This should be a float: there just isn't any compelling reason to pre-judge the required precision. This is not a banking application, nor do we need to worry about the relative speed of integer vs floating point arithmetic.

The name `queued_seconds` is appropriate.